Multisig/UI/Assets/AssetsView.swift

```diff
@@ -12,14 +12,33 @@ struct AssetsView: View {
     @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
     var selectedSafe: FetchedResults<Safe>
 
+    @State var selection: Int? = 0
+
     var body: some View {
-        ZStack {
-            if selectedSafe.first != nil {
-                CoinsView(safe: selectedSafe.first!)
-            } else {
-                AddSafeIntroView()
+        TopTabView($selection) {
+            ZStack {
+                if selectedSafe.first != nil {
+                    BalancesView(safe: selectedSafe.first!)
+                } else {
+                    AddSafeIntroView()
+                }
+            }
+            .gnoTabItem(id: 0) {
+                HStack {
+                    Image("ico-coins")
+                    Text("COINS").font(Font.gnoCaption1)
```

Comment:
Here I could not use CaptionText() as it has 'foregroundColor' that overrides top tabs accent color for selected/not selected states

Reply:
On `iPhone 8 Plus`, `iPhone 11 Pro (13.4.1)`, and `iPhone Pro Max`  the Coins text is truncated. (Ok on `iPhone SE 2nd`, `iPhone 6s`, `iPhone 8`, `iPhone 8 Plus`)

<img src="https://user-images.githubusercontent.com/284056/82440267-93431480-9a8b-11ea-8144-f01cac146471.png" width=300/>

Reply:
for me it works in simulators

Reply:
ok, see the code suggestions that fixes it